js/views/eventLog.js

```diff
@@ -43,6 +43,8 @@ define(["dao", "globals", "ui", "lib/bluebird", "lib/knockout", "util/bbgmView",
                         }
                     });
 
+                    events.map(helpers.correctLinkLid);
```

Comment:
Just want to make sure I follow you... is this because an exported league will have obsolete links in the event log? If so, good thinking!